feat(new-trace): Added static bottom detail panel.

Abdkhan14 opened the pull request:
https://github.com/getsentry/sentry/assets/60121741/9d757d1b-e318-4b87-8d8a-9d91dfef7e0b

Commit:
feat(new-trace): Added static bottom detail panel.

JonasBa commented:
@Abdkhan14 I see a couple opportunities for UX improvements, lets address them later during our call!

Abdkhan14 committed:
feat(new-trace): Made bottom panel resizable. (#66453)

https://github.com/getsentry/sentry/assets/60121741/b4f469af-b88a-4a7d-b4de-480c1efcb93d

---------

Co-authored-by: Abdullah Khan <abdullahkhan@PG9Y57YDXQ.local>
